pkg/storage/image.go

```diff
@@ -86,7 +86,7 @@ type ImageServer interface {
 	ImageStatus(systemContext *types.SystemContext, filter string) (*ImageResult, error)
 	// PrepareImage returns an Image where the config digest can be grabbed
 	// for further analysis. Call Close() on the resulting image.
-	PrepareImage(systemContext *types.SystemContext, imageName string, options *copy.Options) (types.Image, error)
+	PrepareImage(imageName string, options *copy.Options) (types.Image, error)
```

Comment:
All the other methods are taking a context vs this one not taking anymore.

Reply:
yup, it's not needed in this one as it was not in the old `CanPull`